geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloFeatureWriterTest.scala

```diff
@@ -297,7 +297,7 @@ class AccumuloFeatureWriterTest extends Specification with TestWithDataStore wit
       all.map(_.getAttribute("name")) must containTheSameElementsAs(Seq("will", "george", "sue", "bob", "karen"))
 
       // Verify other date range returns nothing
-      val none = fs.getFeatures(ECQL.toFilter("dtg DURING 2013-01-01T00:00:00Z/2013-12-31T00:00:00Z")).features().toSeq
+      val none = fs.getFeatures(ECQL.toFilter("dtg DURING 2013-12-01T00:00:00Z/2013-12-31T00:00:00Z")).features().toSeq
```

Comment:
no, but creating ranges for a whole year makes this test run really slowly - i don't think it makes the test any less relevant by reducing the range to a month.